src/pages/validator/amplifier/integrator-onboarding.mdx

```diff
@@ -0,0 +1,96 @@
+
+## Devnet information
+
+- Devnet RPC: `http://devnet-amplifier.axelar.dev:26657`
+- Devnet LCD (REST API/JSON RPC): `http://devnet-amplifier.axelar.dev:1317`
+- Devnet gRPC: `devnet-amplifier.axelar.dev:9090`
+- Chain ID: `devnet-amplifier`
+- Other info can be found in [`devnet-amplifier.json`](https://github.com/axelarnetwork/axelar-contract-deployments/blob/main/axelar-chains-config/info/devnet-amplifier.json).
+
+## Set up your CosmWasm devnet environment
+
+1. Install the `v0.35.5` axelard binary. You can either:
+    * Download the pre-built [`axelard v0.35.5`](https://github.com/axelarnetwork/axelar-core/releases/tag/v0.35.5) binary from GitHub for your device. You will also need to download 
+    [`wasmvm v1.3.1`](https://github.com/CosmWasm/wasmvm/releases/tag/v1.3.1) `libwasmvm.x86_64.so` (for Linux) or `libwasmvm.dylib` (for Mac) and in either case add the parent folder to `LD_LIBRARY_PATH`, OR: 
+    * Build `axelard` binary directly from the `axelar-core`: 
+        * Clone the [repo](https://github.com/axelarnetwork/axelar-core)
+        * Checkout the `v0.35.5` release tag
+        * Run `make build`
+1. Generate an account by running `axelard keys add wallet --keyring-backend test`.
+1. Fund your devnet account with `uamplifier` tokens:
+    - Join the [Axelar Discord](https://discord.com/invite/axelar).
+    - Get the Developer Role.
+    - Go to the [**#faucet**](https://discord.com/channels/770814806105128977/1002423218772136056/1217885883152334918) channel and submit a request with your account to get 100 test tokens: `!faucet devnet-amplifier [your axelar wallet address]`
+1. Query your account balance by running `axelard q bank balances axelar1suw3fsckht00c5srgtr8p6lhn58e9lxq9yf5th —-node [RPC]`.
+1. Deploy your CosmWasm contract.
+
+## Run `ampd` / `tofnd`
+
+See the [Become an Amplifier Verifier (Worker)](/validator/amplifier/verifier-onboarding) for detailed instructions on how to run ampd and tofnd on your machine.
+
+## Onboard a chain to Amplifier
+
+1. Build all of the deployed contracts, such as the router and service registry, from [`devnet-amplifier.json`](https://github.com/axelarnetwork/axelar-contract-deployments/blob/main/axelar-chains-config/info/devnet-amplifier.json) using the [Cosmwasm `rust-optimizer`](https://github.com/CosmWasm/optimizer). From the root of the repo, run:
+
+    ```bash
+    docker run --rm -v "$(pwd)":/code \
+    --mount type=volume,source="$(basename "$(pwd)")_cache",target=/target \
+    --mount type=volume,source=registry_cache,target=/usr/local/cargo/registry \
+    cosmwasm/optimizer:0.15.0
+    ```
+
+1. Deploy the verifier:
```

Comment:
```suggestion
1. Deploy the voting verifier. You can also instantiate the [existing](https://github.com/axelarnetwork/axelar-contract-deployments/blob/main/axelar-chains-config/info/devnet-amplifier.json#L370) voting verifier contract which has code id `6` instead of deploying a new one:
```